Custom-Video-Player/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Video Player</title>
+  </head>
+  <body>
+    <div class="player">
+      <video class="player__video" src="video/minoni.mp4"></video>
+      <div class="player__controls">
+        <div class="progress">
+          <div class="progress__fill"></div>
+        </div>
+        <button class="player__button toggle">►</button>
```

Comment:
I think the _toggle_ is a part of the _player_.
What about _player__toggle-button_?

Reply:
good idea